### PR TITLE
improve wonky translation

### PR DESCRIPTION
**Pull request by rezemble**
I accidentally opened the German Image docs and the translation struck me as really sketchy, so here's a few polishes

### CODE REVIEW

**Comment by rlenferink** [comment]
/unassign
/cc @mkorbi 
/assign @mkorbi 

(swapping due to time constraints at my side)

**Comment by mkorbi** [comment]
thank you for fixing this up!
/lgtm
/approve

File: content/de/docs/concepts/containers/images.md [hunk 1]
```diff
@@ -299,9 +300,9 @@ resources:
 EOF
 ```
 
-Dies muss für jeden Pod getan werden der eine eigene Registry nutzt.
+Dies muss für jeden Pod getan werden, der eine eigene Registry nutzt.
```
Comment (tanjunchen) [inline content/de/docs/concepts/containers/images.md:303]: /assign
@rezemble
pls see https://git.k8s.io/community/CLA.md#the-contributor-license-agreement

Comment (tanjunchen) [reply content/de/docs/concepts/containers/images.md:303]: /LGTM

**Comment by jimangel** [comment]
@rezemble please sign the CLA. Instructions: https://github.com/kubernetes/website/pull/20826#issuecomment-625173933

**Comment by mkorbi** [comment]
Hi @rezemble just to ping you again :) Could you please sign the CLA, so we can merge the request?

**Comment by rezemble** [comment]
@mkorbi sorry for the delay, I think I now did that? please verify

**Comment by mkorbi** [comment]
@rezemble thank you, looks good
/hold cancel

**Pull request merged**